projects/dotcms-field-elements/src/components/dot-select/readme.md

```diff
@@ -19,14 +19,6 @@
 | `value`           | `value`           |             | `string`  | `undefined` |
 
 
-## Events
```

Comment:
This is a problem... by moving the events out to the util there is no doc of the events our custom elements do.